common-lib/src/main/scala/com/gu/workflow/api/WSUtils.scala

```diff
@@ -12,21 +12,19 @@ trait WSUtils {
 
   protected def buildRequest(path: String): WSRequest = ws.url(s"$apiRoot/$path")
 
-  def deleteRequest(path: String): Future[WSResponse] =
-    buildRequest(path).delete()
+  def deleteRequest(path: String): Future[WSResponse] = buildRequest(path).delete()
 
   def postRequest(path: String, data: Json = Json.Null): Future[WSResponse] =
     buildRequest(path).withHttpHeaders("Content-Type" -> "application/json").post(data.toString())
 
   def putRequest(path: String, data: Json = Json.Null): Future[WSResponse] =
     buildRequest(path).withHttpHeaders("Content-Type" -> "application/json").put(data.toString())
 
-  def getRequest(path: String, params: List[(String, String)] = List.empty, headers: List[(String, String)] = List.empty, timeout: Duration = 2.seconds): Future[WSResponse] = {
```

Comment:
Do we ever want this timeout or is it okay to remove it entirely?

Reply:
I think it's safe to remove it. The [default](https://www.playframework.com/documentation/2.7.x/ScalaWS#Configuring-Timeouts) is 120 seconds.